rdr_service/offline/export_va_workqueue.py

```diff
@@ -0,0 +1,326 @@
+              'Participant Origination',
+              'Genetic Return of Results Consent Status',
+              'Genetic Return of Results Consent Date',
+              'COPE May PPI Survey Complete',
+              'COPE May Date',
+              'COPE June PPI Survey Complete',
+              'COPE June Date',
+              'COPE July PPI Survey Complete',
+              'COPE July Date',
+              'Consent Cohort',
+              'Program Update',
+              'Date of Program Update',
+              'EHR Expiration Status',
+              'Date of EHR Expiration',
+              'Date of First Primary Consent',
+              'Date of First EHR Consent',
+              'Retention Eligible',
+              'Date of Retention Eligibility',
+              'Deceased',
+              'Date of Death',
+              'Date of Approval',
+              'COPE Nov PPI Survey Complete',
+              'COPE Nov PPI Survey Completion Date',
+              'Retention Status',
+              'EHR Data Transfer',
+              'Most Recent EHR Receipt',
+              'Saliva Collection',
+              'COPE Dec PPI Survey Complete',
+              'COPE Dec PPI Survey Completion Date',
+              'COPE Feb PPI Survey Complete',
+              'COPE Feb PPI Survey Completion Date',
+              'Core Participant Minus PM Date',
+              'Summer Minute PPI Survey Complete',
+              'Summer Minute PPI Survey Completion Date',
+              'Fall Minute PPI Survey Complete',
+              'Fall Minute PPI Survey Completion Date',
+              'Digital Health Consent',
+              'Personal & Family Hx PPI Survey Complete',
+              'Personal & Family Hx PPI Survey Completion Date',
+              'SDOH PPI Survey Complete',
+              'SDOH PPI Survey Completion Date',
+              'Winter Minute PPI Survey Complete',
+              'Winter Minute PPI Survey Completion Date',
+              'New Year Minute PPI Survey Complete',
+              'New Year Minute PPI Survey Completion Date'
+              ]
+_INPUT_CSV_TIME_FORMAT_LENGTH = 18
+_CSV_SUFFIX_LENGTH = 4
+INPUT_CSV_TIME_FORMAT = "%Y-%m-%d-%H-%M-%S"
+_MAX_FILE_AGE = datetime.timedelta(days=7)
+FILE_PREFIX = 'va_daily_participant_wq_'
+
+
+def get_workqueue_participants():
+    with ParticipantSummaryDao().session() as session:
+        return session.query(ParticipantSummary
+                            ).join(HPO, HPO.hpoId == ParticipantSummary.hpoId
+                            ).join(Participant, Participant.participantId == ParticipantSummary.participantId
+                            ).filter(HPO.name == 'VA',
+                                     Participant.isTestParticipant != 1,
+                                     # Just filtering on isGhostId != 1 will return no results
+                                     or_(Participant.isGhostId != 1, Participant.isGhostId == None)
+                            ).all()
```

Comment:
We generally follow the practice of the dao's being the database layer for queries. 
Can we move this to the `ParticipantSummaryDao`?

Reply:
I've moved the query into ParticipantSummaryDao